flank-scripts/README.md

```diff
@@ -1,2 +1,78 @@
 # flank-scripts
-This is repository which contains helper scripts while developing flank
+This repository contains helper scripts for developing flank. For now, it contains just release related scripts.
+
+## Build and usage
+
+### Build
+
+To build flank-scripts:
+1. Run script `buildFlankScripts.sh` in `flank-scripts/bash/` directory
+2. Run command `flank-scripts/gradlew clean assemble shadowJar` and manual copy file from `/flank-scripts/build/libs/flankScripts.jar` to `flank-scripts/bash/`
+3. You could always run/build it from Intellij IDEA 
+
+### Usage
+
+Run the script with arguments 
+`flankScripts COMMAND [ARGS]...`
+
+If you need help with available commands or arguments you could always use option `--help`
+
+## Available commands and options
+
+### Release 
+The release process was described in [document](../docs/release_process.md).  
+To show all available commands for release use:
+`flankScripts release`
+
+Available commands are:
+  - `releaseFlank`      Release Flank on GitHub  
+  - `updateBugsnag`     Update Bugnsag  
+  - `jFrogDelete`       Delete old version on bintray  
+  - `jFrogSync`         Sync maven repository using jfrog  
+  - `deleteOldRelease`  Delete old release on github  
+  - `removeOldTag`      Remove old tag on GitHub  
+  
+#### `releaseFlank`
+Release Flank on GitHub 
+
+Options:  
+  `--input-file`&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Path to release file  
```

Comment:
instead of `&nbsp;`, consider using a GitHub Flavored Markdown Table

Reply:
Done 👍